### ISSUE TITLE
Add [unique] for non-PK foreign key

### ISSUE THREAD

**Issue by ttngu207** [datajoint/datajoint-python#656]
A very simple example case:

```
@schema
class TableA(dj.Manual):
    definition = """
    a_attr: int
    """
```

```
@schema
class TableB(dj.Manual):
    definition = """
    b1_attr: int
    b2_attr: int
    ---
    b3_attr: int
    """
```

and 

```
@schema
class TableC(dj.Manual):
    definition = """
    -> TableA
    ---
    -> [unique] TableB
    """
```

but currently, such definition for 'TableC' is not supported, instead, I need to do something like:
```
@schema
class TableC(dj.Manual):
    definition = """
    -> TableA
    ---
    -> TableB
    unique index (b1_attr, b2_attr)
    """
```

**Comment by guzman-raphael** [comment]
@ttngu207 Quick question on this. Doesn't the primary key definition of `TableB` imply that these are already unique? Or do you mean that they should be unique between `b1_attr` and `b2_attr`. As in the below are valid entries for `TableB`:

b1_attr   b2_attr   b3_attr
1	1	7
1	2	3
2	2	5
2	9	6

However, for `TableC`, rows 1 and 3 would not be a valid insert?

**Comment by dimitri-yatsenko** [comment]
This is issue #301, which should have been implemented with #495. I will test to see why it didn't work.

**Comment by dimitri-yatsenko** [comment]
This was a typo in the SQL code that only showed for composite unique foreign keys
